README.md

```diff
@@ -54,42 +54,13 @@ See [Architecture](./doc/ARCHITECTURE.md) for the architecture of [**pgagroal**]
 * [FreeBSD](https://www.freebsd.org/)
 * [OpenBSD](http://www.openbsd.org/)
 
-## Compiling the source
 
-[**pgagroal**](https://github.com/agroal/pgagroal) requires
+## Compiling from sources
 
-* [gcc 8+](https://gcc.gnu.org) (C17)
-* [cmake](https://cmake.org)
-* [make](https://www.gnu.org/software/make/)
-* [libev](http://software.schmorp.de/pkg/libev.html)
-* [OpenSSL](http://www.openssl.org/)
-* [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
-* [rst2man](https://docutils.sourceforge.io/)
-* [libatomic](https://gcc.gnu.org/wiki/Atomic)
-* [cJSON](https://github.com/DaveGamble/cJSON)
+[**pgagroal**](https://github.com/agroal/pgagroal) can be compiled from sources,
+after having installed all the required dependencies.
+See the [documentation about installing the required dependencies](doc/DISTRIBUTIONS.md).
```

Comment:
We should list the main dependencies here such that people have an idea of the general architecture